scripts/compile_tests/failures_histogram.py

```diff
@@ -149,6 +146,5 @@ def as_issue(count, msg, repro, tests=None):
         action="store_true",
     )
     args = parser.parse_args()
-    failures_histogram(
-        args.eager_dir, args.dynamo_dir, args.verbose, args.format_issues
-    )
+    dynamo38, dynamo311, eager311 = download_reports(args.commit)
```

Comment:
do we not need eager38? There are likely tests that are skipped on eager3.8 that are also skipped on dynamo3.8 (esp. numpy ones), but pass on eager3.11.

Reply:
I'll figure out how to do this in a follow-up. The failures_histogram.py script only worked on a single python version before, but now that we have a way to more easily download reports we should actually do something with the python 3.8 information.